### PR TITLE
Define ANSI style at Uid level, update ocamlformat

### PR DESCRIPTION
**Pull request by AltGr**

### CODE REVIEW

File: compiler/desugared/linting.ml [hunk 1]
```diff
@@ -35,8 +35,8 @@ let detect_empty_definitions (p : program) : unit =
           then
             Message.emit_spanned_warning
               (ScopeDef.get_position scope_def_key)
-              "In scope @{<yellow>\"%a\"@}, the variable @{<yellow>\"%a\"@} is \
-               declared but never defined; did you forget something?"
+              "In scope \"%a\", the variable \"%a\" is declared but never \
```
Comment (denismerigoux) [inline compiler/desugared/linting.ml:38]: I love it

**Review by denismerigoux — APPROVED** [review]
So bumping `ocamlformat` didn't change anything? I'm impressed

File: compiler/desugared/linting.ml (hunk 1 above)
Comment (AltGr) [reply compiler/desugared/linting.ml:38]: Maybe I put *too many* colors in there but now it's very easy to tweak them in `shared_ast/definitions.ml` ; feel free to tweak it if you find readability is impaired.

**Comment by AltGr** [comment]
> So bumping `ocamlformat` didn't change anything? I'm impressed

Hum actually no there was a slightly hidden error that said that since the version wasn't the one expected no reformatting had been redone :unamused:
But changes aren't that big this time (and clearly an improvement overall)

**Pull request merged**